src/flows/templates/types/demo-data

```diff
@@ -0,0 +1,199 @@
+          "labels": "/api/v2/buckets/04944ca2401c3558/labels",
+          "owners": "/api/v2/buckets/04944ca2401c3558/owners",
+          "members": "/api/v2/buckets/04944ca2401c3558/members"
+        },
+        "orgID": "e859306b862b3e21",
+        "labels": [],
+        "createdAt": "2019-12-17T15:46:51.584984427Z",
+        "updatedAt": "2019-12-17T15:46:51.584984546Z",
+        "retentionRules": [
+          {
+            "type": "expire",
+            "everySeconds": 259200
+          }
+        ],
+        "readableRetention": " 3 days"
+      },
+      "measurement": "average_temperature",
+      "aggregateFunction": {
+        "name": "mean"
+      },
+      "title": "Select Input Metric",
+      "visible": true
+    },
+    {
+      "id": "local_e9e821bc-745e-4ae3-a4be-3c72f3155670",
+      "type": "visualization",
+      "period": "1d2h24m10s",
+      "functions": [
+        {
+          "name": "mean"
+        }
+      ],
+      "properties": {
+        "type": "simple-table",
+        "showAll": false
+      },
+      "panelHeight": 200,
+      "panelVisibility": "visible",
+      "title": "Visualize average_temperature",
+      "visible": true
+    },
+    {
+      "id": "local_bda92ec7-3a42-4b45-8901-452b833c4e1c",
+      "mode": "preview",
+      "text": "One of the most common use cases for InfluxDB is downsampling data to reduce the overall disk usage as data collects over time.\n\nThis is done by aggregating data within windows of time, then storing the aggregate value in a new bucket.",
+      "type": "markdown",
+      "title": "Step 3 - Let's Downsample",
+      "visible": true
+    },
+    {
+      "id": "local_12a7e386-5504-453b-98a2-f773e39632dd",
+      "type": "rawFluxEditor",
+      "queries": [
+        {
+          "text": "// __Previous_Result__ uses data from all the panels above it as an input to the Flux function\n__PREVIOUS_RESULT__\n\n  // This Flux function windows and aggregates the data in to 15m averages\n|> aggregateWindow(fn: mean, every: 1h)",
+          "editMode": "advanced",
+          "builderConfig": {
+            "tags": [],
+            "buckets": [],
+            "functions": []
+          }
+        }
+      ],
+      "activeQuery": 0,
+      "title": "Downsample with Flux",
+      "visible": true
+    },
+    {
+      "id": "local_0d8184c3-5c36-48d2-be0e-58bc06a0b6c5",
+      "mode": "preview",
+      "text": "You might want to keep your downsampled data separate from your raw data with a longer retention policy.\n\nSelect a separate bucket or create a new one from the dropdown in the next output cell.\n\nThis generates:\n\n`|> to(bucket: \"YOUR NEW BUCKET\")`\n\nBonus: You can switch the notebook mode from Preview to Run to write this data to the new bucket manually.\n",
+      "type": "markdown",
+      "title": "Step 4 - Select Output",
+      "visible": true
+    },
+    {
+      "id": "local_fe73a34e-52d8-4a64-86fa-8e846af7d0d9",
+      "type": "toBucket",
+      "bucket": {
```

Comment:
if you really want to export to the `_tasks` bucket, delete everything in this object outside of `name`. i'd suggest just setting it to `null` instead of telling users to put it in a weird place